met-api/src/met_api/services/engagement_content_service.py

```diff
@@ -36,10 +38,29 @@ def create_engagement_content(engagement_content_data, engagement_id):
         sort_index = EngagementContentService._find_higest_sort_index(engagement_id)
 
         engagement_content_data['sort_index'] = sort_index + 1
+
         created_content = EngagementContentService._create_content(engagement_id, engagement_content_data)
         created_content.commit()
+
+        if engagement_content_data.get('content_type') == EngagementContentType.Custom.name:
+            EngagementContentService.create_default_custom_content(engagement_id, created_content.id)
+
         return EngagementContentSchema().dump(created_content)
 
+    @staticmethod
+    def create_default_custom_content(eng_id: int, eng_content_id: int):
+        """Create default engagement custom content."""
+        default_summary_content = {
+            'engagement_id': eng_id
+        }
+        try:
+            EngagementCustomContentService.create_custom_content(eng_content_id, default_summary_content)
+        except Exception as exc:  # noqa: B902
```

Comment:
What does "# noqa: B902" mean here?

Reply:
The linter suggests adding specific exceptions, but I've opted for a general exception handling approach to ensure that any error encountered during the creation of custom content raises a business exception error.